tests/system/test_pandas.py

```diff
@@ -112,6 +188,21 @@ def test_load_table_from_dataframe_w_automatic_schema(bigquery_client, dataset_i
         bigquery.SchemaField("uint8_col", "INTEGER"),
         bigquery.SchemaField("uint16_col", "INTEGER"),
         bigquery.SchemaField("uint32_col", "INTEGER"),
+        bigquery.SchemaField("array_bool_col", "BOOLEAN", mode="REPEATED"),
+        bigquery.SchemaField("array_ts_col", "TIMESTAMP", mode="REPEATED"),
+        # BigQuery does not support uploading DATETIME values from
+        # Parquet files. See:
+        # https://github.com/googleapis/google-cloud-python/issues/9996
```

Comment:
FYI: This actually has been fixed in the backend, you just have to explicitly ask for DATETIME in the schema when loading the dataframe to avoid breaking changes. https://github.com/googleapis/python-bigquery/pull/895

I've filed https://github.com/googleapis/python-bigquery/issues/985 to address this in google-cloud-bigquery v3.

Let's add a TODO here pointing at https://github.com/googleapis/python-bigquery/issues/985 instead.

Reply:
Done.